lua/ulx/modules/sh/sh_cfc_restart_ulx.lua

```diff
@@ -0,0 +1,41 @@
+require( "cfc_restart_lib" )
+local CATEGORY_NAME = "Utility"
+
+if SERVER then
+    util.AddNetworkString( "CFC_RESTART_START" )
+    util.AddNetworkString( "CFC_RESTART_STOP" )
+end
+
+local function cfcRestartTimerCallback()
+    timer.Destroy( "CFC_RESTART_TIMER" )
+    
+    local restarter = CFCRestartLib()
+    restarter:restart()
+end
+
+local function cfcUlxRestart( calling_ply, delay )
+    net.Start( "CFC_RESTART_START" )
+    net.WriteInt( math.Round( delay ),  16 )
+    net.Broadcast()
+
+
```

Comment:
One of these newlines could go, while spacing out the code is a good thing, too much spacing can look weird :)